runtime/js/40_command.js

```diff
@@ -0,0 +1,214 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+"use strict";
+
+((window) => {
+  const core = window.Deno.core;
+  const { pathFromURL } = window.__bootstrap.util;
+  const { read, write } = window.__bootstrap.io;
+  const { illegalConstructorKey } = window.__bootstrap.webUtil;
+  const { ArrayPrototypeMap, ObjectEntries, String, TypeError } =
+    window.__bootstrap.primordials;
+
+  function createWritableIOStream(rid) {
+    return new WritableStream({
+      async write(chunk) {
+        let nwritten = 0;
+        while (nwritten < chunk.byteLength) {
+          nwritten += await write(rid, chunk.subarray(nwritten));
+        }
+      },
+      abort() {
+        core.tryClose(rid);
+      },
+    });
+  }
+
+  function createReadableIOStream(rid) {
+    return new ReadableStream({
+      async pull(controller) {
+        const view = controller.byobRequest.view;
+        const res = await read(rid, view);
+        if (res === null) {
+          core.close(rid);
+          controller.close();
+        } else {
+          controller.byobRequest.respond(res);
+        }
+      },
+      cancel() {
+        core.close(rid);
+      },
+      type: "bytes",
+      autoAllocateChunkSize: 16384,
+    });
+  }
+
+  class Command {
+    #rid;
+
+    constructor(command, {
+      args = [],
+      cwd = undefined,
+      clearEnv = false,
+      env = {},
+      uid = undefined,
+      gid = undefined,
+    } = {}) {
+      this.#rid = core.opSync("op_create_command", {
+        cmd: pathFromURL(command),
+        args: ArrayPrototypeMap(args, String),
+        cwd: pathFromURL(cwd),
+        clearEnv,
+        env: ObjectEntries(env),
+        uid,
+        gid,
+      });
+    }
+
+    spawn(options = {}) {
+      if (this.#rid === null) {
+        throw new TypeError("Child has already been used up.");
```

Comment:
Sorta confusing error message. Maybe something more similar to this?
```suggestion
        throw new TypeError("Child process has already terminated.");
```
Also applies to other places with the same error message (I count at least 4.)